samples/openapi3/client/petstore/python-experimental/petstore_api/models/user.py

```diff
@@ -136,6 +144,10 @@ def __init__(self, _check_type=True, _from_server=False, _path_to_item=(), _conf
             password (str): [optional]  # noqa: E501
             phone (str): [optional]  # noqa: E501
             user_status (int): User Status. [optional]  # noqa: E501
+            arbitrary_object (bool, date, datetime, dict, float, int, list, str): test code generation for objects Value must be a map of strings to values. It cannot be the &#39;null&#39; value.. [optional]  # noqa: E501
```

Comment:
@spacether , these allowed types would have to be fixed in a separate PR.

Reply:
Yup, this weekend I am busy but I can get to it in the future